src/shipit_static_analysis/requirements.txt

```diff
@@ -3,3 +3,10 @@
 libmozdata
 RBTools
 PyYAML
+
+# Parsepatch before release
```

Comment:
What does "Parsepatch before release" mean here?

Reply:
[Parsepatch library](https://github.com/mozilla/parsepatch) is not yet published on Pypi.
I'll need to update this once it gets stable & published.

Reply:
Thanks! Same remark, a `FIXME` would work great here (but feel free to merge faster by not changing the code).

Reply:
I prefer to open an issue: #729